tests/requirements.txt

```diff
@@ -0,0 +1 @@
+scikit-learn==0.21.3
```

Comment:
Again, please remove/merge to main requirements.txt file.

Reply:
Done. Note that running the build from the PR now requires the image at mcr.microsoft.com/mlops/python:latest to have been updated accordingy.